app/views/core/RootComponent.coffee

```diff
@@ -17,13 +17,15 @@ module.exports = class RootComponent extends RootView
 
       @vueComponent = new @VueComponent({
         el: @$el.find('#site-content-area')[0]
+        propsData: @propsData
```

Comment:
Add this property to this class along with `VueComponent` and `vuexModule` as null, for reference.